src/server/admin/teams/controllers/save/edit-team.js

```diff
@@ -18,7 +18,8 @@ const editTeamController = {
       const { response } = await editTeam(request, cdpTeam.teamId, {
         name: cdpTeam.name,
         description: cdpTeam.description,
-        github: cdpTeam.github
+        github: cdpTeam.github,
+        ...(cdpTeam.serviceCode && { serviceCodes: [cdpTeam.serviceCode] })
```

Comment:
How does the User Service handle this? feels like it would be good to always send `serviceCodes` but if there isn't one then just send an empty array?

```
serviceCodes: cdpTeam.serviceCode ? [cdpTeam.serviceCode] : []

```